package.json

```diff
@@ -114,6 +118,7 @@
     "phantomjs-prebuilt": "2.1.4",
     "react-addons-test-utils": "0.14.3",
     "react-hot-loader": "1.3.0",
+    "react-imageloader": "2.1.0",
```

Comment:
ehh image loader should definitely be in prod deps... those that are in devDeps are either only used for tests or in the webpack process (like react-hot-loader)